pages/common/copyq.md

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
Link can be updated to https://copyq.readthedocs.io/en/latest/command-line.html here.

Reply:
Fixed. :)